xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -478,7 +478,8 @@
         #set ($disabledButton = "disabled")
         #set ($titleMessageTranslationKey = "like.button.title")
       #end
-      &lt;div class="like-button btn $btnClass $disabledButton badge" title="$escapetool.xml($services.localization.render($titleMessageTranslationKey, [$likeCount]))"&gt;
+      &lt;button class="like-button btn $btnClass $disabledButton badge"
```

Comment:
I think the button should be explicitly disabled when liking isn't possible, e.g., because the user isn't logged in.

Reply:
Okay, setting the HTML attribute `disabled` when needed. :+1: 
___
On another hand, I realized the title wasn't correct as it did not convey the _purpose_ of the button.
I think this is a similar case as the discussion we had here:
https://github.com/xwiki/xwiki-platform/pull/2051#discussion_r1086365978
I'll change the title so it displays both the current state (amount of likes) and the action that would happen when clicking.
e.g. 
`Like the page. Current number of likes on this page: {0}`
`Unlike the page. Current number of likes on this page: {0}`
`Can't like the page. Current number of likes on this page: {0}`